client/src/store/selectors/widgets/loginPage/index.ts

```diff
@@ -0,0 +1,5 @@
+import type { State } from "../../../reducers";
+
+export const getIsLoading = (state: State) => state.widgets.loginPage.isLoading;
+export const getErrorMessage = (state: State) =>
+  state.widgets.loginPage.errorMessage;
```

Comment:
Получение виджета удобнее вынести в отдельный селектор. Эта часть уже сейчас повторяется в двух селекторах и будет повторяться в каждом последующем при добавлении новых полей в виджете

```suggestion
export const getLoginPageWidget = (state:  State) => state.widgets.loginPage;

export const getIsLoading = (state: State) => getLoginPageWidget(state).isLoading;

export const getErrorMessage = (state: State) => getLoginPageWidget(state).errorMessage;
```

Reply:
исправил